### ISSUE TITLE
UI feedback on Airwindows Consolidated; increase font size

### ISSUE THREAD

**Issue by redactedscribe** [baconpaul/airwin2rack#76]
Hello,

Thank you for the consolidation, it's much more convenient this way. However, the UI isn't really too good, and I'm not talking aesthetics (although, those could be improved too). Some basic UI tweaks to improve functionality:

- Increase the font size; text is much too small. Scaling the window (if it were possible) wouldn't be the right solution here to gain better clarity. Bigger default font so that the text is more legible without any additional zooming or scaling would be best. You have a scrollbar for the description area so no need to use so small text here. Even the description's header text is slightly too small. The font chosen may also be part of the legibility issue, but increasing the size will surely help:
    - I suggest making the description area's font 1-2pts larger than its header's current font size, and the header itself 2-4pts larger, if I had to guess.
    - Also, the name of the knobs should be more prominent and easily read as they are the things you interact with primarily.
- The Airwindows logo should ask for confirmation when clicking it whether to open the browser or not. It's never great when you launch your browser accidentally via a plugin.
- Without additional layout changes, the menu system probably can't have its font size increased (it's already quite long), unless it begins to scroll when it reaches the screen's limits. If that is the case, a slight bump to font size here would help too.

In short, a bump to the font size would help the design and functionality of this great collection of plugins.

Thanks again.

![Bitwig_Studio_2024-05-07_01-08-23](https://github.com/baconpaul/airwin2rack/assets/25013746/30563ab8-4d1e-47a8-ac3c-1c0f037682a1)

**Comment by baconpaul** [comment]
Sigh windows font rendering in JUCE can be such garbage. Here's how it looked on the mac where I developed it.

I'll run some low DPI windows 11 later this week and try and get something reasonable. thanks for the feedback.

![Screenshot 2024-05-06 at 6 35 59 PM](https://github.com/baconpaul/airwin2rack/assets/13387561/55ae9278-8794-4d70-8a3f-6426a0dce067)

**Comment by baconpaul** [comment]
oh and good idea on the prompt-to-open-brtowser on the airwindows click. I'll add that too.

**Comment by baconpaul** [comment]
OK thanks to some conversations with @jatinchowdhury18 and a few other audio devs, and jatin helping me fix the font metrics, and me doing some code refactoring, we now have this situation much improved.

Here's my mac on a 4k retina display:

![image](https://github.com/baconpaul/airwin2rack/assets/13387561/3ad30d73-48d6-4ed1-9f54-bf743f4d45a9)


Here's Windows 11 on 100% (so no HDPI)

![image](https://github.com/baconpaul/airwin2rack/assets/13387561/10a87941-0f00-46a8-9a3f-fece3cf4114d)


And here's Windows 11 on 150% (so "some" HDPI)

![image](https://github.com/baconpaul/airwin2rack/assets/13387561/a4ea6b35-efa6-416c-81a3-f095884c9683)


This clearly resolves the basic problem. I'm waiting for my CI build to run but i'll merge it and when I do I'll close this issue (adding a small straggler for the prompt-to-url issue) and a new plugin should be available in an hour or so.

Thank you for raising this. I didn't realize how mis-rendered Windows was until your screenshots! Appreciated.

**Comment by baconpaul** [comment]
OK this is merged; there should be a new plugin ready for installation in about 30-40 minutes. 

Thanks again for the detailed report.

**Comment by baconpaul** [comment]
The build is now complete and the situation is greatly improved. Please do give it a try at your convenience and let us know how it works for you.

**Comment by baconpaul** [comment]
Great. Thanks for flagging and glad we fixed it.

To answer your question: No plugin has more than 10 parameters and the vertical space is set up so 10 parameters would have uniform spacing between the picker and footer elements.
